CHANGELOG

```diff
@@ -21,6 +21,10 @@ Enhancements
   * add timer for block-IO and block-compute
   * store block information in `_block` attribute (Issue #89)
   * add parallel density class (Issue #8)
+  * add `cdf` attribute to `pmda.rdf.InterRDF` for cumulative
+    distribution function
+  * deprecate `get_cdf` method in `pmda.rdf.InterRDF_s` and use
```

Comment:
Put this in a separate category *Changes*
```reST
* The method rdf.InterRDF_s.get_cdf() was removed; just use the managed attribute
  rdf.InterRDF_s.cdf
```

Reply:
In MDAnalysis we would tag it with a deprecation and keep it around for a while. In PMDA we can move a bit faster.